src/components/Error.tsx

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React, { useContext } from 'react';
+import classNames from 'classnames';
```

Comment:
Would be better to do it in short way
```suggestion
import cn from 'classnames';
```